src/App.js

```diff
@@ -1,56 +1,25 @@
 import React from 'react';
-import logo from './logo.svg';
-import { Counter } from './features/counter/Counter';
+import { Provider } from 'react-redux';
+import { Routes, Route } from 'react-router-dom';
+import store from './redux/store';
+import Header from './components/Header';
+import Continent from './components/Continent';
+import CountryDetails from './components/CountryDetails';
 import './App.css';
 
 function App() {
```

Comment:
- [ ] For most of the part your app follows the design guidelines, however kindly consider adding images tour countries so your app looks like the given design

|Your App|Provided Design|
|-|-|
|![Screenshot_20221008_183612](https://user-images.githubusercontent.com/25227022/194720396-43884933-533d-4dfb-a3ef-b4237ac578f6.png)|![Screenshot_20221008_183840](https://user-images.githubusercontent.com/25227022/194720422-73981c6e-ad5c-401e-88bb-247d458a9f55.png)|